test/testutil/tu_local.h

```diff
@@ -12,6 +12,8 @@
 #include <openssl/bio.h>
 #include "../testutil.h"
 
+#define TEST_SKIP_CODE  123
```

Comment:
I wonder if this return value could produce some side effect at some point :)

Reply:
It's possible but unlikely.  The value could vary between OSSL versions if it starts being abused.